Dockerfile

```diff
@@ -10,7 +10,7 @@ USER hmcts
 
 # ---- Build image ----
 FROM base as build
-RUN PUPPETEER_SKIP_DOWNLOAD=true yarn install
+RUN PUPPETEER_SKIP_CHROMIUM_DOWNLOAD=true yarn install
```

Comment:
was this change pushed as  part of hmcts/cmc-citizen-frontend#3394?

Reply:
it was part of #3395.